runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ValidateRunnerXlangTest.java

```diff
@@ -54,15 +60,36 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** Test External transforms. */
+/**
+ * Runner Validation Test Suite for Cross-language Transforms.
+ *
+ * <p>As per Beams's Portability Framework design, Cross-language transforms should work out of the
```

Comment:
I think this can be replaced by something like following which gives a more direct explanation:

Cross-language transforms runner validation test suite for Java SDK.

These tests check whether core Beam transforms work correctly when used as cross-language transforms from Java SDK. This is needed to make sure that cross-language transforms framework works correctly for various corner cases. Please see documentation for individual tests for more details regarding cases covered by each test. Please see here<link to doc> for more details.

Reply:
+1 for more direct and concise explanation.

Reply:
I've made it more explicit now.